Assets/Dependencies/SmashBrew/Character/States/MovementState.cs

```diff
@@ -142,7 +142,7 @@ public enum CharacterFacingMode {
             get { return _ledgeTarget; }
         }
 
-        public Transform CurrentLedge {
+        public GameObject CurrentLedge {
             get { return _currentLedge; }
             set {
                 bool grabbed = _currentLedge == null && value != null;
```

Comment:
Now that this is a syncrhonized variable, this needs to be guarded by a `isServer` check.